compair/api/answer.py

```diff
@@ -294,6 +294,9 @@ def post(self, course_uuid, assignment_uuid, answer_uuid):
         require(EDIT, answer)
         restrict_user = not allow(MANAGE, assignment)
 
+        if current_app.config.get('DEMO_INSTALLATION', False) and assignment.id in [1,2] and answer.user_id in range(3, 33):
```

Comment:
Added class variables in DemoDataFixture with the static ids and rewrote all these type of conditions